### PR TITLE
Travis/docker

### PR DESCRIPTION
**Pull request by Yuhanun**
Changed the travis builds to now use a docker image with pre-built dependencies.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,63 +1,41 @@
...
+  - docker
+
+# Disable git cloning cause we do that in docker
+git:
+  clone: false
+
+# This happens before your build
+before_install:
+  # Pull the docker image, not cached, caching is slower
+  - docker pull yuhanuncitgez/roboteamtwente:latest
```
Comment (rolfvdhulst) [inline .travis.yml]: Does this not call your fork instead of ours? Or is this docker magic I don't know of ?

File: .travis.yml [hunk 2]
```diff
@@ -1,63 +1,41 @@
...
-  - cd roboteam_ai            # ai has to be on development for some qt fixes
-  - git checkout development  # ai has to be on development for some qt fixes
-  - cd ..                     # ai has to be on development for some qt fixes
-  - mv ${TRAVIS_BUILD_DIR} .
-  - mkdir cmake-build-debug && cd cmake-build-debug;
-  - cmake -DCMAKE_BUILD_TYPE=Debug -G "CodeBlocks - Ninja" ..
-  - cmake --build . --target utils_tests -- -j 8
-  - cd roboteam_utils
-  - ./utils_tests
+    - ~/.ccache
```
Comment (rolfvdhulst) [inline .travis.yml]: No tests? I'd really like to see them run here.

Comment (Yuhanun) [reply .travis.yml]: Tests are ran inside roboteam_ai.sh, check roboteam_suite, those are contained in the docker image. it does test.

File: .travis.yml (hunk 1 above)
Comment (Yuhanun) [reply .travis.yml]: docker pull pulls from

hub.docker.com, not from git, we need to set up a hub account for roboteam.

**Pull request merged**